yang/frr-bfdd.yang

```diff
@@ -219,7 +219,7 @@ module frr-bfdd {
 
     leaf required-echo-receive-interval {
       type uint32 {
-          range "0 | 10000..max";
+          range "0..10000 | 10000..max";
       }
```

Comment:
is there really a meaning (and is it feasible to do so in software) to have so low values?